test/torch/tensors/test_gc.py

```diff
@@ -196,20 +199,18 @@ def test_implicit_garbage_collect_logging_on_pointer(workers):
 
 
 def test_websocket_garbage_collection(hook, start_proc):
-    # Args for initializing the websocket server and client
-    base_kwargs = {"id": "ws_gc", "host": "localhost", "port": 8555, "hook": hook}
-    server = start_proc(WebsocketServerWorker, **base_kwargs)
-
+    kwargs = {"id": "ws_gc", "host": "localhost", "port": 8555, "hook": hook}
+    process_remote = start_proc(WebsocketServerWorker, **kwargs)
     time.sleep(0.1)
-    client_worker = WebsocketClientWorker(**base_kwargs)
+    local_worker = WebsocketClientWorker(**kwargs)
 
     sample_data = torch.tensor([1, 2, 3, 4])
-    sample_ptr = sample_data.send(client_worker)
+    sample_ptr = sample_data.send(local_worker)
 
     _ = sample_ptr.get()
-    assert sample_data not in client_worker._objects
+    assert sample_data not in local_worker._objects
 
-    client_worker.ws.shutdown()
-    client_worker.ws.close()
+    local_worker.ws.shutdown()
+    local_worker.ws.close()
```

Comment:
I think local_worker.close() should be sufficient (replacing both local_worker.ws.shutdown() and local_worker.ws.close())